lib/router.js

```diff
@@ -1421,6 +1421,10 @@ router.get('/grandchallenge/challenges', lazyloadRouteHandler('./routes/grandcha
 // 西北工业大学
 router.get('/nwpu/:column', lazyloadRouteHandler('./routes/nwpu/index'));
 
+// 西北民族大学
```

Comment:
There are huge warnings at both the beginning and the end of this file.

https://github.com/DIYgod/RSSHub/blob/c678cb6c6156563180eacd27d1509e5438df2dc9/lib/router.js#L18 https://github.com/DIYgod/RSSHub/blob/c678cb6c6156563180eacd27d1509e5438df2dc9/lib/router.js#L1792